cmd/nknd/nknd.go

```diff
@@ -502,16 +504,26 @@ func main() {
 }
 
 func GetID(seeds []string, publickey []byte) ([]byte, error) {
-	id, err := chain.DefaultLedger.Store.GetID(publickey)
```

Comment:
It's better to do the following logic:
1. Check if AllowGetID1 at local height, if not, use the old logic to avoid seed node dependency
2. If AllowGetID1 is true at local height, then get id from seed node (no need to get height first because it's the same amount of rpc call)